concreep-refilled/locale/en/locale.cfg

```diff
@@ -10,6 +10,7 @@ cover landfill=Cover landfill
 debug=Various debug output (very spammy)
 debug function calls=Function call debug output (very spammy)
 debug coroutine calls=Coroutine call debug output (very spammy)
+hypercreep=enables hypercreep mode (world eater)

```

Comment:
agreed

Reply:
what about changing the mod name to paved paradise? :D kidding here however it would make it more clear that this fork of conreep will have a major functionality upgrade